dashboard/app/models/sections/section.rb

```diff
@@ -81,7 +81,29 @@ def sti_name
   alias_attribute :lesson_extras, :stage_extras
 
   validates :participant_type, acceptance: {accept: Curriculum::SharedCourseConstants::PARTICIPANT_AUDIENCE.to_h.values, message: 'must be facilitator, teacher, or student'}
-  validates :grade, acceptance: {accept: [SharedConstants::STUDENT_GRADE_LEVELS, SharedConstants::PL_GRADE_VALUE].flatten, message: "must be one of the valid student grades. Expected one of: #{[SharedConstants::STUDENT_GRADE_LEVELS, SharedConstants::PL_GRADE_VALUE].flatten}. Got: \"%{value}\"."}
+
+  # Custom CSV serializer for the grades array.
+  class GradesArray
```

Comment:
not sure if there's a better place to put this class?

Reply:
I'm not sure I love this being in the middle of the `validate` code. I'd probably either put it at the bottom of this class or split it out into a separate file and add it to https://github.com/code-dot-org/code-dot-org/tree/staging/dashboard/lib.

Reply:
I moved it to `lib`, because anywhere else I put it in this class or below it, it didn't load soon enough and I just got a NameError when I tried to use it. it didn't seem to fit in `lib/services` or `lib/policies` so i just put it directly in `lib`.

Reply:
Sounds good thank you!